pkg/template/apis/template/types.go

```diff
@@ -108,10 +108,24 @@ type TemplateInstanceSpec struct {
 // TemplateInstanceRequester holds the identity of an agent requesting a
 // template instantiation.
 type TemplateInstanceRequester struct {
-	// Username is the username of the agent requesting a template instantiation.
+	// The name that uniquely identifies this user among all active users.
```

Comment:
I think convention here is to start with a lower case character, and I got it wrong previously in this file.  Please can you fix up the ones you're changing/adding and we can table the rest for some other time?